subsys/net/lib/shell/net_shell_private.h

```diff
@@ -8,20 +8,40 @@
 #include <zephyr/shell/shell.h>
 #include <zephyr/net/net_ip.h>
 
-#define PR(fmt, ...)						\
-	shell_fprintf(sh, SHELL_NORMAL, fmt, ##__VA_ARGS__)
+#define PR(fmt, ...)					\
```

Comment:
Either keep all of the backslashes aligned within the macro, or just drop the extra backslash indentation. I don't see a point of having some of them aligned, and some not.

Reply:
Then I just let `.clang-format` decide for me 😄